### ISSUE TITLE
Pixie related code still in Genny

### ISSUE THREAD

**Issue by chrisheller** [treeform/genny#21]
Just took a quick perusal of this.  Looks great.  

I did notice that there are a few places where there is some Pixie related code still hanging around though.  For example,  in the exportObjectNim proc in genny/languages/nim.nim,  there is a hard-coded check for some Pixie related types (Vector2, Matrix3, etc.)

genny/languages/python.nim also has this (e.g. mapping Vec2 to Vector2 in exportTypePy). 

I'm guessing that is just from the re-factoring to split Genny out of Pixie and into its own project though.

**Comment by guzba** [comment]
Yep that is correct. We're removing hard-coded bits like this as we add a means to accomplish the same thing within the Genny DSL. We still need to sort out some type name re-mapping stuff and also errors (PixieError is hardcoded iirc). These will be sorted soon.

**Comment by treeform** [comment]
I have removed the Pixie-specific hardcodes from Genny’s generator code. Genny now derives these bindings from the binding files, including tricky external/value types such as Vec2 and Mat3 from vmath, plus simpler types like Rect and Color. Pixie is still used extensively as a test fixture, but the generator itself no longer has Pixie-specific code paths.

https://github.com/treeform/genny/pull/67
